app/services/ServiceAccount.scala

```diff
@@ -0,0 +1,34 @@
+package services
+
+import com.google.auth.Credentials
+import com.google.auth.oauth2.ServiceAccountCredentials
+import com.google.cloud.bigquery.{BigQuery, BigQueryOptions}
+
+import java.io.{ByteArrayInputStream, InputStream}
+
+object ServiceAccount {
+
+  def bigQuery(credentials: Credentials, projectId: String): BigQuery =
+     BigQueryOptions
```

Comment:
most of this file is unused. I suggest moving the relevant part into `BigQueryService.apply` -
```
val bigQueryOptions = BigQueryOptions
  .newBuilder()
  .setCredentials(credentials)
  .setProjectId(projectId)
  .build()
  .getService
new BigQueryService(bigQueryOptions)
```